Wikipedia/UI-V5/WMFHomeViewController.m

```diff
@@ -427,6 +435,9 @@ - (void)controller:(id<WMFHomeSectionController>)controller didSetItems:(NSArray
 
 - (void)controller:(id<WMFHomeSectionController>)controller didAppendItems:(NSArray*)items {
     NSInteger section = [self indexForSectionController:controller];
+    if (section == NSNotFound) {
```

Comment:
what about setting the delegate to `nil` until after it's loaded? also, i don't understand how loading a section is asynchronous. adding items to the collectionView is (since we're using `performBatchUpdates:completion:`), but isn't the `sections` property of the `SSSectionedDataSource` updated instantly?